coa_flask_app/events.py

```diff
@@ -0,0 +1,158 @@
+"""
+A module handle the logic with the event table.
+"""
+
+from datetime import datetime
+from typing import List, Tuple
+
+from coa_flask_app.db_accessor import Accessor
+
+
+Event = Tuple[int, int, int, int, float, float]
+
+
+def get(volunteer_year: int, volunteer_season: str) -> List[Event]:
+    """
+    Gets a list of events.
+
+    Returns:
+        A list of events.
+    """
+    query = """
+            SELECT
+                event_id,
+                site_id,
+                volunteer_cnt,
+                trashbag_cnt,
+                trash_weight,
+                walking_distance
+            FROM coa_data.event AS cde
+            WHERE
+                cde.volunteer_year = %s AND
+                cde.volunteer_season = %s
+            """
+    with Accessor() as db_handle:
+        db_handle.execute(query, (volunteer_year, volunteer_season))
+        return db_handle.fetchall()
+
+
+def add(
+    updated_by: str,
+    site_id: int,
+    volunteer_year: int,
+    volunteer_season: str,
+    volunteer_cnt: int,
+    trashbag_cnt: int,
+    trash_weight: float,
+    walking_distance: float,
+) -> None:
+    """
+    Adds an item.
+
+    Args:
+        updated_by: The user adding the item.
+        site_id: The ID of the site where the event took place.
+        volunteer_year: The year of event.
+        volunteer_season: The season of the event.
+        volunteer_cnt: The count of volunteers at the event.
+        trashbag_cnt: The count of trashbags collected.
+        trash_weight: The weight of the trashbags.
+        walking_distance: The total distance walked of the volunteers.
+    """
+    mon = 4 if volunteer_season == "Fall" else 10
```

Comment:
I just realized we used title case for the season instead of all caps in the trigger, which would maybe be better? I think it's fine as is though.

But this should be "mon = 4 if volunteer_season == "Spring"...". I'd actually recommend validating that the season is "Spring" or "Fall" in the add / update requests.